examples/complete/main.tf

```diff
@@ -111,6 +111,45 @@ module "cbr_zone_schematics" {
   }]
 }
 
+#############################################################################
+# Create EN instance, destination, topic and subscription
```

Comment:
I suggest to move this block under the `event_notification` module block since it will happen after instance is cretaed